disco_aws_automation/version.py

```diff
@@ -1,5 +1,5 @@
 """Place of record for the package version"""
 
-__version__ = "2.3.8"
+__version__ = "2.3.9"
```

Comment:
This will break existing scripts/jobs that are using sandboxes. This should be a bigger version jump. Also this will require coordination with the other teams to update their sandbox scripts. Are you going to take care of that?

Reply:
you mean version update or removing hardcoded word?

Reply:
currently different teams have different path to their sandboxes e.g.
https://github.com/amplifylitco/asiaq_arrested_config/tree/master/sandbox/configs
https://github.com/amplifylitco/asiaq_dataservices_config/tree/master/sandbox

the question: why they still can use script with hardcoded value `sandboxes` as part of path, or they use some alternatives?